src/Microsoft.DotNet.VersionTools/Automation/LocalVersionsRepoUpdater.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.VersionTools.Automation
+{
+    public class LocalVersionsRepoUpdater : VersionsRepoUpdater
+    {
+        /// <summary>
+        /// Updates only the Latest_Packages file in the specified on-disk versions repository dir.
+        /// </summary>
+        public void UpdateBuildInfoLatestPackages(
+            IEnumerable<string> packagePaths,
+            string versionsRepoDir,
+            string versionsRepoPath)
```

Comment:
IMO if I saw this signature in intellisense without seeing the implementation I would be confused.